src/Action/ViewPostAction.php

```diff
@@ -84,4 +95,15 @@ public function setSeoPage(SeoPageInterface $seoPage = null)
     {
         $this->seoPage = $seoPage;
     }
+
+    /**
+     * @return bool
+     */
+    protected function isVisible(PostInterface $post)
+    {
+        // TODO: move to sonata
```

Comment:
Uuuu… this is Sonata

Reply:
If it is for him he should add `TODO` to PR body if not, we need more explanation and we use `NEXT_MAJOR` instead of `TODO`, right?